Automattic/wp-calypso
# Jetpack Cloud: replace the oauth landing page with a redirection

#### Changes proposed in this Pull Request

* Remove the OAuth landing page and replace it with a redirection to `/oauth2/authorize`.
* Redirect to `https://jetpack.com` after a log out.

#### Questions

* Should we add these updates to the `jetpack-cloud-horizon.json` configuration file?
* If I go to `http://jetpack.cloud.localhost:3000/scan/some-site` the `/scan/some-site` part will get lost in the middle of the flow. In other words, after logging in I won't be redirected back to `/scan/my-site`. Should we create a separate task for this?

#### Testing instructions

* Open an incognito window (any browser)
* Go to `http://jetpack.cloud.localhost:3000/`
* You should be immediately redirected to `https://wordpress.com/log-in` (no more landing OAuth page)
* Log in with your credentials
* Once you're back in Jetpack Cloud, log out from click on your avatar (mastebar)
* You should be redirected to `https://jetpack.com`

Fixes 1169345694087188-as-1177457980972593

#### Demo
![Kapture 2020-05-26 at 16 40 11](https://user-images.githubusercontent.com/3418513/82943845-9b5df100-9f70-11ea-9aab-beccf0c9ab8c.gif)
